website/docs/d/managed_application_definition.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+subcategory: "ManagedApplication"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_application_definition"
+description: |-
+  Gets information about an existing Managed Application Definition
+---
+
+# Data Source: azurerm_managed_application_definition
+
+Uses this data source to access information about an existing Managed Application Definition.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_managed_application_definition" "example" {
+  name                = "example-managedappdef"
+  resource_group_name = "example-resources"
+}
+
+output "id" {
+  value = data.azurerm_managed_application_definition.existing.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Managed Application Definition.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where this Managed Application Definition exists.
+
+## Attributes Reference
```

Comment:
You should at least mention `id` here. Regarding to the other attributes here, are they meant to be used by any other resources?

Reply:
updated